### DISCUSSION TITLE
Quick Start Guide

### DISCUSSION THREAD

**Discussion by agorosti** [HCL-TECH-SOFTWARE/connections-automation#83] [Q&A]
Hi,

Since I never played before with ansible, I am following the quick-start-guide,  and I would setup a testing enviroment exactly with the same vms machines and hostnames provided there.    Could you please clarify the minimal requirements on AWS for such as machines ? Although the guide already mention the CPUs and the Memory,   I  am particularly concern about disk space (a minimal one, for a POC),  I setup the machines as stated on the quick-guide, but the standar disk sizes that come with AWS are small (only 10GB), I guess that is NOT enough,  I would like to known aproximately how disk space I need on each machine.

Can you please suggest about volumes to use / add  to the EC2 instances ?

Thks !!

**Comment by dejo1307** [comment]
Hi Andres,

From what we are doing all around in AWS, I don't think you would need ever more then 100G on any machine, and even those 100G in case you are setting up single node with everything on it and to count on space that you will clean up in the end when you remove all the downloaded zip files and stuff. 

In a smallest two nodes environments we never use more then 50G for one node which is WAS and all for WAS and 100G for CP side (simply to buy ourselves some time with Docker caching images etc since we are doing also constant upgrades). If you plan to play with it in a way we do - constantly recreating different scenarios (recycling infrastructure) 50/100 is a good combination. If you plan to keep it longer, 100/100 is good combination. 

If you are making fully distributed environment, you need even less on proxy (we have 20G there and even that is a lot), DMGR, IHSs also 20. If you are going to host NFS on DMGR then you put 50-100 there, depending how much data you plan to keep there.

But in general, 100 is if you really want to forget about it for a long time, anything more about that is really not needed for non production environments. 

For two nodes environments, we use for example mostly m5a.xlarge for the WAS/Connections/DB2 instance, and m5a.4xlarge for Kubernetes/CP instance, but that one works fine also with m5a.2xlarge.

Best regards,
Dejan

**Reply by agorosti** [reply]
Thks a lot !!!  I am willing to start the  install !!! :-)
